unlock-app/src/components/creator/lock/AppStore.js

```diff
@@ -153,6 +153,24 @@ const AppStore = ({ lock }) => {
   return (
     <Wrapper>
       <Details>
+        {
+          lock.network === 100 || lock.network === 134 || lock.network === 4 // show on rinkeby for debug
```

Comment:
Ok, let's actually move this to `CreatorLock`. Also not all locks deployed on xdai or polygon are affected. only those for which the `unlock https://github.com/unlock-protocol/unlock/blob/c3f55c616f45e13dbcfd6a944ed5d3304d9cea2c/unlock-app/src/components/creator/CreatorLock.js#L117